TeamSnapSDK/SDK/DataTypes/TSDKEvent.m

```diff
@@ -159,5 +158,16 @@ - (NSString *)displayNameWithOpponent:(TSDKOpponent *)opponent preferShortLabel:
     }
 }
 
+- (TSDKRepeatingEventTypeCode)repeatingTypeCode {
+    if ([[self.collection data] objectForKey:@"repeating_event_type_code"]) {
```

Comment:
We need a specific getter for this since the default of NIL/NSNotFound isn't helpful!